.gitignore

```diff
@@ -0,0 +1,163 @@
+data/*.csv
```

Comment:
Re-roll from master and this file should disappear (this file isn't a change from this commit).